pages/viewPlans/index.js

```diff
@@ -1,6 +1,6 @@
 import { useAllPlans } from "@/utils/helpers";
 import { useRouter } from "next/router";
-import Link from "next/link";
+import TemplateCard from "@/components/TemplateCard/TemplateCard";
```

Comment:
You might want to reconsider the naming convention of your component folders. 

instead of `components/TemplateCard/TemplateCard.js`

you could go for `components/TemplateCard/index.js`

then the import would be

``` js
import TemplateCard from "@/components/TemplateCard";
```

Reply:
Thanks for the hint. 
I might consider this down the road for refactoring.